### ISSUE TITLE
Поддержка

### ISSUE THREAD

**Issue by CyberValenok** [wpcraft-ru/wooms#336]
Ребятки, я все понимаю, но вы продукт продаете, можно чтоб хоть кто-то отреагировал. Единственное место где есть какая-то активность, так это гитхаб. Это круто, конечно, но по оплате тут не место обсуждать

**Comment by l0stmagic** [comment]
Согласен. Это единственное место которое живет. Мне очень необходим функционал с обменом статусов, который на данный момент сыпет ошибками в логе.

**Comment by CyberValenok** [comment]
На wpcraft на почты и формы никто не отвечает, как с вами связываться, мб телега разработчиков, или еще что-то. Надо по покупке обсудить, просто пока вы не отвечаете, отвечает конкурент с более плохим продуктом. Ребятки, проявите активность, я заполнил форму на сайте, написал на почту на сайте, написал одному из участников репозитория, у которого нашел данные и нигде ответа нет. Куда писать?
